node/subsystem-util/src/runtime/mod.rs

```diff
@@ -154,19 +182,19 @@ impl RuntimeInfo {
 	}
 
 	/// Convenience function for checking the signature of something signed.
-	pub async fn check_signature<Context, Payload, RealPayload>(
+	pub async fn check_signature<Sender, Payload, RealPayload>(
 		&mut self,
-		ctx: &mut Context,
+		sender: &mut Sender,
```

Comment:
sidenote: This is a good idea, but iirc you will need the context to spawn new messages - at least in the long run - the `ToOverseer::Spawn*` should be removed.